src/pages/success-stories/sharkdao.page.tsx

```diff
@@ -0,0 +1,126 @@
+import { Trans, t } from '@lingui/macro'
+import { CaseStudiesConfigurationProps } from 'components/CaseStudies/CaseStudiesConfiguration'
+import { CaseStudiesHeader } from 'components/CaseStudies/CaseStudiesHeader'
+import { CaseStudyContentContainer } from 'components/CaseStudies/CaseStudyContentContainer'
+import ExternalLink from 'components/ExternalLink'
+import SectionHeader from 'components/SectionHeader'
+import { AppWrapper } from 'components/common'
+import { CASE_STUDY_PROJECTS } from 'constants/successStoryProjects'
+import Image from 'next/image'
+
+export default function SharkDAOPage() {
+  const sharkDAOSuccessStoryProject = CASE_STUDY_PROJECTS[2]
```

Comment:
Risky - if we change the case_study_project array, this will break

Reply:
Worth changing do you think?

Reply:
eh. meh